v1/feed.yaml

```diff
@@ -97,3 +97,182 @@ paths:
+  problem:
+    required:
+      - type
+    properties:
+      type:
+        type: string
+      title:
+        type: string
+      detail:
+        type: string
+      instance:
+        type: string
+
+  thumbnail:
+    type: object
+    properties:
+      source:
+        type: string
+        description: Thumbnail image URI
+      width:
+        type: integer
+        description: Thumbnail width
+      height:
+        type: integer
+        description: Thumnail height
+    required: ['source', 'width', 'height']
+
+  article_summary:
+    type: object
+    properties:
+      title:
+        type: string
+        description: Article title in a form of DB key
+      normalizedtitle:
+        type: string
+        description: Article title as it should be presented to the user
+      thumbnail:
+        description: Lead image for the article
+        $ref: '#/definitions/thumbnail'
+      description:
+        type: string
+        description: Wikidata description of the article
+      extract:
+        type: string
+        description: First several sentences of an article in plain text
+    required:
+      - title
+
+  mostread_article:
+    type: object
+    properties:
+      title:
+        type: string
+        description: Article title in a form of DB key
+      normalizedtitle:
+        type: string
+        description: Article title as it should be presented to the user
+      views:
+        type: integer
+        description: Number of views on the requested day
+      rank:
+        type: integer
+        description: Position in the list of most viewed articles
+      fromencoded:
+        type: boolean
+      thumbnail:
+        description: Lead image for the article
+        $ref: '#/definitions/thumbnail'
+      description:
+        type: string
+        description: Wikidata description of the article
+      extract:
+        type: string
+        description: First several sentences of an article in plain text
+    required:
+      - title
+      - normalizedtitle
+      - views
+      - rank
+      - pageid
+
+  mostread:
+    type: object
+    properties:
+      date:
+        type: string
+        description: The date which the data correspond to
+      articles:
+        type: array
+        description: Array of most popular articles
+        items:
+          $ref: '#/definitions/mostread_article'
+    required:
+      - date
+      - articles
+
+  news_item:
+    type: object
+    properties:
+      story:
+        type: string
+        description: A cover story for the news item
+      links:
+        type: array
+        description: A collection of articles related to the news item
+        items:
+          $ref: '#/definitions/article_summary'
+    required:
+      - story
+      - links
+
+  news:
+    type: array
+    items:
+      $ref: '#/definitions/news_item'
+
+  random:
+    type: object
+    properties:
+      items:
+        type: array
+        items:
```

Comment:
@gwicke using items here would be in line with other RB endpoints and would allow adding more random articles without paint. 

If you use different keys, how would you add 5 random articles instead of 1? `random`, `random2`, `random3` ... ? Kinda ugly. Since it doesn't really matter let's leave it as is?